specification/security/put_role/SecurityPutRoleRequest.ts

```diff
@@ -29,13 +29,14 @@ import { RequestBase } from '@_types/Base'
 import { Metadata, Name, Refresh } from '@_types/common'
 
 /**
- * Create or update roles API.
- *
- * Create or update roles in the native realm.
+ * Create or update roles.
+ * The role management APIs are generally the preferred way to manage roles in the native realm, rather than using file-based role management.
+ * The create or update roles API cannot update roles that are defined in roles files.
```

Comment:
Could we also add a note here that file-based role management is not available in Serverless? It's a bit confusing otherwise.

Reply:
Sure! Added in 304227bbd6cc1e6360b82264ada0dc409ca932f8